pkg/monitor/api.go

```diff
@@ -91,16 +92,47 @@ func StartKubeAPIMonitoring(ctx context.Context, m *Monitor, clusterConfig *rest
 }
 
 func StartOpenShiftAPIMonitoring(ctx context.Context, m *Monitor, clusterConfig *rest.Config, timeout time.Duration) error {
+	if err := StartOpenShiftAPIMonitoringWithGET(ctx, m, clusterConfig, timeout); err != nil {
+		return err
+	}
+	if err := StartOpenShiftAPIMonitoringWithPATCH(ctx, m, clusterConfig, timeout); err != nil {
+		return err
+	}
+	return nil
+}
+
+func StartOpenShiftAPIMonitoringWithGET(ctx context.Context, m *Monitor, clusterConfig *rest.Config, timeout time.Duration) error {
 	pollingConfig := *clusterConfig
 	pollingConfig.Timeout = timeout
 	pollingClient, err := clientimagev1.NewForConfig(&pollingConfig)
 	if err != nil {
 		return err
 	}
 
+	return startOpenShiftAPIMonitoringFor(ctx, m, "GET", func() error {
```

Comment:
I think we'll need to split for another case.  This test is conflating
1. openshift-apiserver is running
2. kube-apiserver can reach openshift-apiserver over network
3. openshift-apiserver can contact etcd

Each different case requires different debugging and implies different problems and inspection by different teams.  We can at least distinguish between 2 and 3 from outside the cluster and we should attempt to do so.  You can do this by hitting the discovery endpoint with GET and PATCH in addition to hitting am imagestream.